ci/run.sh

```diff
@@ -11,7 +11,6 @@ rustfmt -vV
 cargo build --locked -v --release --target "$TARGET" --features vendored-openssl
 
 if [ -z "$SKIP_TESTS" ]; then
-  cargo test --release -p download --target "$TARGET" --features vendored-openssl
```

Comment:
Why remove the tests for the download crate?

Reply:
My mistake